uamqp/_decode.py

```diff
@@ -261,12 +264,15 @@ def decode_payload(buffer):
             except KeyError:
                 message["data"] = [value]
         elif descriptor == 118:
-            message["sequence"] = value
+            try:
+                message["sequence"].append(value)
+            except KeyError:
+                message["sequence"] = [value]
         elif descriptor == 119:
             message["value"] = value
         elif descriptor == 120:
             message["footer"] = value
-    return message
+    return Message(**message)
```

Comment:
I had avoided this (along with `Properties` and `Headers` above) in decode - Not sure it provides much value for the cost. Shouldn't be needed for any of the SDKs.

Reply:
I agree, it's not necessary as it's private.
I just feel that it's a asymmetry in sending and receiving + type hint (ivar vs a key in a dict), so made the change.

did you avoid this for perf concern?